openjdk.test.jck/src/test.jck/net/adoptopenjdk/stf/Jck.java

```diff
@@ -325,7 +325,7 @@ public void execute(StfCoreExtension test) throws Exception {
 				 jckVersion.contains("jck9") ||
 				 jckVersion.contains("jck8")) &&
 				 (testSuite == TestSuite.RUNTIME) ) {
-				executeJar = "/lib/jtlite.jar" ;
+				executeJar = "/lib/javatest.jar" ;
```

Comment:
Could we just simple remove this if-else block if javatest.jar works for all jckVersion? (Though I'm not sure the original idea about using jtlite.jar for 8+ and javatest.jar for 7-.

Reply:
Yes! Updated.